apps/tng-dictionary/src/components/TermsDetail/TermsDetail.tsx

```diff
@@ -11,95 +11,91 @@ import TermData, { Term } from '../Term/Term';
 import './TermsDetail.module.css';
 
 /* eslint-disable-next-line */
-export interface TermsDetailComponentProps { }
+export interface TermsDetailComponentProps {}
 
 type ComponentState = {
-  termData: null | TermData;
-}
+    termData: null | TermData;
+};
 
 export function TermsDetailComponent(props: TermsDetailComponentProps) {
-
```

Comment:
We should minimize the amount of jsx we write in one place. It seems git gives up on the diffs when you have a giant chunk of jsx.